.github/workflows/locale_update_all.yaml

```diff
@@ -26,16 +26,23 @@ jobs:
       RAILS_ENV: i18n
       SKIP_TEST_RESET: true
       SKIP_DATABASE_SETUP: true
+      BUNDLER_GROUPS: "manageiq_default,ui_dependencies,qpid_proton"
     steps:
     - uses: actions/checkout@v4
     - name: Set up system
       run: bin/before_install
+    - name: Install qpid dependency - libcurl4-openssl-dev
+      run: sudo apt-get update && sudo apt-get install -y libcurl4-openssl-dev
+    - name: Install qpid dependency - libqpid-proton11-dev
+      run: sudo apt-add-repository --yes 'deb http://us.archive.ubuntu.com/ubuntu lunar universe' && sudo apt-get install -t lunar -y libqpid-proton11-dev
```

Comment:
This should already be in the `bin/before_install` - if not, that s where these belong - There were some recent changes for Ubuntu Lunar in other plugins, so maybe that same change needs to happen here?  cc @agrare

Reply:
We only need this because `locale:update_all` ends up calling [eager_load!](https://github.com/grosser/gettext_i18n_rails/blob/df42959a673837f6bbd5d003af97b1dd2dab0d41/lib/gettext_i18n_rails/model_attributes_finder.rb#L70) from the call to [gettext:store_model_attributes](https://github.com/ManageIQ/manageiq/blob/82e2139209b50c8616b382c349f914a9fa231311/lib/tasks/locale.rake#L143).  Providers don't need qpid_proton.  Neither does core.

Reply:
Just had a thought, are we going to have to do the same with the systemd gems that are based on ruby-dbus?  Those are in an optional group because of Mac as well.

Reply:
We can continue to ignore the file that can't be required without qpid_proton gem installed and loaded.

```
    message_handler_path = Pathname.new(Vmdb::Plugins.paths[ManageIQ::Providers::Nuage::Engine]).join("app/models/manageiq/providers/nuage/network_manager/event_catcher/messaging_handler.rb")
    Rails.autoloaders.main.ignore(message_handler_path)
```

I was trying to remove it but I guess we can keep it to avoid this shotgun surgery.

Reply:
For now, I'll drop the qpid dependency and unconditionally tell zeitwerk to ignore eager loading the file that depends on the library at load time.